internal/db/db_blueprints.go

```diff
@@ -4,11 +4,18 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"time"
 
 	"github.com/google/uuid"
 	"github.com/jackc/pgx/v4"
 )
 
+type ComposeEntryWithVersion struct {
+	ComposeEntry
+	BlueprintId      uuid.UUID
+	BlueprintVersion int
+}
```

Comment:
This is fine, slight disadvantage is what happens when you add another (few) fields from the other table. It will no longer be `ComposeEntryWithVersion` :-) Maybe can you remove `Entry` from the type name? To me "entry" means it is a literal record (entry) while this is not.